File: Java/(Beecrowd)1060 - Positive Numbers.java
import java.util.Scanner;

public class Main{
    
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        
        int posit = 0;
        for (int i=0; i<6; i++){
            float x = input.nextFloat();
            
            if (x > 0){
                posit += 1;
            }
        }
        
        System.out.println(posit + " valores positivos");
    }
    
}

File: C/(Beecrowd)1060 - Positive Numbers.c
#include <stdio.h>

int main() {
    float num; 
    int posit = 0;

    for (int i = 0; i < 6; i++) {
        scanf("%f", &num);
        if (num > 0) {
            posit ++;
        }
    }

    printf("%d valores positivos\n", posit);

    return 0;
}